src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackResultsPageUiTest.java

```diff
@@ -97,7 +98,31 @@ public void testContent() {
         resultsPage = loginToInstructorFeedbackResultsPage("CFResultsUiT.instr", "Empty Session");
         resultsPage.waitForPanelsToCollapse();
         resultsPage.verifyHtmlMainContent("/instructorFeedbackResultsPageEmpty.html");
+        
+    }
+    
+    @Test
+    public void testExceptionalCases() {
+        ______TS("Case where more than 1 question with same question number");
+        // results page should be able to load incorrect data and display it gracefully
+        
+        FeedbackQuestionAttributes question = testData.feedbackQuestions.get("qn2InSession4");
+        assertEquals(2, question.questionNumber);
+        // need to retrieve question from datastore to get its questionId
+        FeedbackQuestionAttributes questionFromDatastore = 
+                                        BackDoor.getFeedbackQuestion(question.courseId, 
+                                                                     question.feedbackSessionName, 
+                                                                     question.questionNumber);
+        assertEquals(question, questionFromDatastore);
+        // make question have the same question number as another question
+        questionFromDatastore.questionNumber = 1;
+        
+        BackDoor.editFeedbackQuestion(questionFromDatastore);
 
+        resultsPage = loginToInstructorFeedbackResultsPage("CFResultsUiT.instr", "Session with errors");
+        
+        resultsPage.waitForPanelsToCollapse();
+        resultsPage.verifyHtmlMainContent("/instructorFeedbackResultsPageWithDuplicateQuestionNumber.html");
```

Comment:
Oops, this is failing sometimes. The id of each question panel is still determined by sort order.